### DISCUSSION TITLE
Checkboxes side by side and not on top of each other

### DISCUSSION THREAD

**Discussion by gerhardmpl** [Budibase/budibase#4687] [Help] [answered]
Is it possible to align checkboxes in a form / field group side by side and aligned with the input field like in this mock up? This are individual boolean columns in a `bbdb` table. 

![grafik](https://user-images.githubusercontent.com/40722408/155607281-c6f15197-64d5-4fbe-b353-e0dc43b651b8.png)

**Comment by mjashanks** [comment, marked as answer]
Haven't tried this out fully, but you could try put checkboxes inside a container like so:

![image](https://user-images.githubusercontent.com/3524181/155609882-2898e71e-84a5-4698-8c1b-aecf082cffe9.png)

You'll have to 
1. Give each checkbox its own custom `Field` name
2. IN the `Save Row` action, use the "Add Column" to manually use these fields. You might need javascript here.

![image](https://user-images.githubusercontent.com/3524181/155610138-88343e25-611a-40ee-869f-2075e5685478.png)

**Comment by gerhardmpl** [comment]
This! The trick is to have the labels `above` in the field group, as can be seen in this screen (not a mock up). Works straight with the tables `boolean` fields. Putting the labels `left` leads to problems in aligning with the other fields (which was my initial problem).

![grafik](https://user-images.githubusercontent.com/40722408/155615527-680e0859-c833-4b92-85e6-f547e2fb20e7.png)

Thank you!
